src/components/Search/SearchForm.jsx

```diff
@@ -64,6 +64,7 @@ class SearchForm extends React.Component {
       <form onSubmit={this.formSubmit} style={{ width: small ? '280px' : 'auto' }}>
         <TextField
           id="searchField"
+          aria-label={strings.search_title}
```

Comment:
Visual users have the hint text to go off of, but because this field lacks an associated label, screen reader users currently only hear `edit text blank`. With this aria label screen reader users now have the same context visual users have. 

<details><summary>Current</summary>
<img width="611" alt="Screen Shot 2021-10-02 at 10 24 51 AM" src="https://user-images.githubusercontent.com/15097156/135722838-d98db639-6643-4666-a72d-df757f50c969.png">
</details>

<details><summary>With fix</summary>
<img width="661" alt="Screen Shot 2021-10-02 at 9 43 08 AM" src="https://user-images.githubusercontent.com/15097156/135722868-bf8f7c46-85f5-451a-a0fa-35582462f04a.png">
</details>